src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -594,8 +594,9 @@ protected DynamoModel(IStartConfiguration config)
                 PreferenceSettings = settings;
                 PreferenceSettings.PropertyChanged += PreferenceSettings_PropertyChanged;
             }
-            // If user does not agree to GA terms, do not try to launch the client at all
-            if (PreferenceSettings.IsAnalyticsReportingApproved)
+            // If user does not agree to GA or ADP terms, do not try to launch the client at all
+            if (PreferenceSettings.IsAnalyticsReportingApproved ||
+                PreferenceSettings.IsADPAnalyticsReportingApproved)
```

Comment:
This is GA instrumentation here, will be removed later and should only be tied to GA agreement. We may want to revert the change here

Reply:
changed code to use configuration setting.